homeassistant/components/mqtt/strings.json

```diff
@@ -1,4 +1,10 @@
 {
+  "issues": {
+    "encoding_b64": {
+      "title": "Configuration issue for MQTT camera",
+      "description": "The `encoding` option was configured to set the image encoding to Base64. This is deprecated, use the `image_encoding` option instead."
```

Comment:
Let's avoid the term "deprecated" in the UI. Instead, tell it is "going to be removed".

Reply:
I discussed with @emontnemery, we will skip issue reporting for this PR (so for now).